archivist/archivist.py

```diff
@@ -112,6 +113,11 @@ def __init__(self, url, *, auth=None, cert=None, verify=True):
         self._locations = None
         self._attachments = None
 
+        self.assets: Optional[_AssetsClient]
+        self.events: Optional[_EventsClient]
+        self.locations: Optional[_LocationsClient]
+        self.attachments: Optional[_AttachmentsClient]
```

Comment:
I believe that this may change in 3.10 - is that correct?
If so add a comment so that we do something about it when we decide to support 3.10